core/common/src/main/java/alluxio/wire/BackupStatus.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import alluxio.AlluxioURI;
+import alluxio.exception.AlluxioException;
+import alluxio.grpc.BackupPStatus;
+import alluxio.grpc.BackupState;
+
+import com.google.common.base.MoreObjects;
+import com.google.protobuf.ByteString;
+import org.apache.commons.lang.SerializationUtils;
+
+/**
+ * Defines the status of a backup.
+ */
+public class BackupStatus {
+  /** State of backup. */
```

Comment:
Should there be an unique identifier for the backup?

Reply:
I was planning on getting unique IDs after uncovering some client side issue.
When run concurrently, backup shell commands could start picking up results for runs that they haven't launched.

Reply:
https://github.com/Alluxio/alluxio/pull/10182/commits/2ed4f85af8905341a0b972c38f62d49e921f3ce9